wikipron/cli.py

```diff
@@ -24,15 +24,29 @@ def _get_cli_args(args: List[str]) -> argparse.Namespace:
     )
     parser.add_argument(
         "--stress",
+        action="store_true",
+        dest="stress",
         default=True,
-        help="By default, stress marks are included in pronunciations."
-        "To remove them, set this flag to False.",
+        help="Include stress marks in pronunciations.",
+    )
+    parser.add_argument(
+        "--no-stress",
+        action="store_false",
+        dest="stress",
+        help="Remove stress marks from pronunciations.",
     )
     parser.add_argument(
         "--syllable-boundaries",
+        action="store_true",
+        dest="syllable_boundaries",
```

Comment:
I think this one is also redundant because it maps `--syllable-boundaries` onto `args.syllable_boundaries`?